components/Spaces/Dashboard/SpaceCardModal.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import {
+  Container,
+  Box,
+  Typography,
+  Dialog,
+  List,
+  ListItem,
+  ListItemText,
+  ListItemAvatar,
+  Avatar,
+} from "@material-ui/core";
+
+import { useTheme } from "@material-ui/core/styles";
+
+import PersonIcon from "@material-ui/icons/Person";
+
+
+
+function UserList(props) {
+  const { users } = props;
+  return (
+    <List>
+      {users.map((user) => (
+        <ListItem key={user.name} className="py-0">
+          <ListItemAvatar>
+            <Avatar>
+              <PersonIcon />
+            </Avatar>
+          </ListItemAvatar>
+          <ListItemText primary={user.name} secondary={user.status} />
+        </ListItem>
+      ))}
+    </List>
+  );
+}
+
+export default function SpaceCardModal(props) {
+  const theme = useTheme();
+  const { onClose, open, children, friends, participants, hosts } = props;
+  
+
+  const handleClose = () => {
+    onClose();
+  };
+
```

Comment:
don't need it's own function. Just do `<Dialog onClose={() => onClose()} />`
onClose isn't a very good function name either. `handleClose` would be more appropriate